### ISSUE TITLE
Add rule to remove useless awaits used with expect

### ISSUE THREAD

**Issue by rafiazman** [playwright-community/eslint-plugin-playwright#306]
Playwright assertions (https://playwright.dev/docs/test-assertions) are split into auto-retrying assertions and non-retrying assertions.

A new rule can be created perhaps with the name `no-useless-await-expect` that would disallow unnecessary awaits for Playwright expects.

Example of incorrect code:
```js
await expect(true).toBeTruthy()
```

Example of correct code:
```js
expect(true).toBeTruthy()
```

**Comment by antebudimir** [comment]
+1 for this rule.

**Comment by antebudimir** [comment]
Thanks for the effort! 🍻
